website/docs/r/linux_virtual_machine_scale_set.html.markdown

```diff
@@ -238,6 +238,12 @@ A `data_disk` block supports the following:
 
 -> **NOTE:** `UltraSSD_LRS` is only supported when `ultra_ssd_enabled` within the `additional_capabilities` block is enabled.
 
+* `disk_encryption_set_id` - (Optional) The ID of the Disk Encryption Set which should be used to encrypt this Data Disk.
+
+-> **NOTE:** The Disk Encryption Set must have the `Reader` Role Assignment scoped on the Key Vault - in addition to an Access Policy to the Key Vault
+
+~> **NOTE:** Disk Encryption Sets are in Public Preview in a limited set of regions
```

Comment:
Should we list the regions here?

Reply:
nope since this is dynamic/OOB (it's gone GA in some reasons, but not everywhere yet)